src/account.ts

```diff
@@ -31,7 +31,10 @@ export type TransactionRequest = {
   maxFeePerGas?: string | number | bigint | BigNumber | ArrayLike<number>;
 };
 
-export async function createAccount(options: CreateAccountOptions = {}) {
+async function _createAccountWithMnemonicPromise(
```

Comment:
Ah, that "Promise" isn't referring to the return type of the function but to the first param (since "WithMnemonic" felt to me like it'd take a string directly) - but agree it's fine to drop anyway since I ended up calling the first arg "mnemonicPromise" anyway)